### PR TITLE
feat(DENG-351): added model for fxa_users_services_devices_first_seen

### PR DESCRIPTION
**Pull request by kik-kik**
# feat(DENG-351): added model for fxa_users_services_devices_first_seen

Added a new model for FxA data which contains records of when a specific user, service, device was first observed.

### CODE REVIEW

File: dags/bqetl_fxa_events.py [hunk 1]
```diff
@@ -351,6 +351,21 @@
         arguments=["--schema_update_option=ALLOW_FIELD_ADDITION"],
     )
 
+    ox_accounts_derived__fxa_users_services_devices_first_seen__v1 = bigquery_etl_query(
```
Comment (gkabbz) [inline dags/bqetl_fxa_events.py:354]:
```suggestion
   fx_accounts_derived__fxa_users_services_devices_first_seen__v1 = bigquery_etl_query(
```

Comment (kik-kik) [reply dags/bqetl_fxa_events.py:354]: This is autogenerated, this is fine.

File: dags/bqetl_fxa_events.py [hunk 2]
```diff
@@ -390,3 +405,7 @@
     firefox_accounts_derived__fxa_users_services_devices_daily__v1.set_upstream(
         firefox_accounts_derived__fxa_content_events__v1
     )
+
+    ox_accounts_derived__fxa_users_services_devices_first_seen__v1.set_upstream(
```
[the remark this answers is not in the document]
Comment (kik-kik) [reply dags/bqetl_fxa_events.py:409]: autogenerated by `./bqetl  dag generate` can't change this.

File: sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_devices_first_seen_v1/schema.yaml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+- mode: NULLABLE
+  name: os_name
+  type: STRING
+  description:
+
+- mode: NULLABLE
+  name: country
+  type: STRING
+  description:
```
Comment (gkabbz) [inline sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_devices_first_seen_v1/schema.yaml:50]: Missing description.

File: sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_devices_first_seen_v1/schema.yaml [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  name: language
+  type: STRING
+  description:
+
+- mode: NULLABLE
+  name: entrypoint
+  type: STRING
+  description: |
+    The entrypoint of the flow session.
+    Typically a UI touchpoint like "preferences".
```
Comment (gkabbz) [inline sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_devices_first_seen_v1/schema.yaml:62]:
```suggestion
   How a user begins their registration or login flow. Typically a UI touchpoint like "preferences" or ...
```

File: dags/bqetl_fxa_events.py [hunk 5]
```diff
@@ -351,6 +351,21 @@
         arguments=["--schema_update_option=ALLOW_FIELD_ADDITION"],
     )
 
+    ox_accounts_derived__fxa_users_services_devices_first_seen__v1 = bigquery_etl_query(
+        task_id="ox_accounts_derived__fxa_users_services_devices_first_seen__v1",
```
[the remark this answers is not in the document]
Comment (kik-kik) [reply dags/bqetl_fxa_events.py:355]: autogenerated, can't change this.

**Pull request merged**